### ISSUE TITLE
Could not use `merlin_synthesis.sh` script on osX

### ISSUE THREAD

**Issue by maelp** [CSTR-Edinburgh/merlin#93]
I trained the model in `egs/slt_arctic` using `run_full_voice.sh`, then installed `festival` and `festival_speech_tools` in `tools/`, copied a voice in `festival/lib/voices`, then added a `utts.data` file with sample text data in `experiments/slt_arctic_full/test_synthesis`, but I get an error (festival seems to be crashing)

Could you tell me if there is another step to do (I assume I don't have to create `.lab` files manually, it is done automatically)? Could you give an example of `utts.data` in the proper format to make sure I used the correct one

What is weird is that the `run_full_voice.sh` training + synthesizing worked out-of-the-box (after my modifications for osX https://github.com/CSTR-Edinburgh/merlin/pull/92) and could generate synthesized wav files, but for some reasons the `merlin_synthesis.sh` script requires me to install festival, and crashes when running

**Comment by ronanki** [comment]
run_full_voice.sh script uses existing data to train the models and synthesizes files only for given test data. 

In the same folder, the text data from slt_arctic is available at:
[slt_arctic_demo_data/cmuarctic.data
](http://festvox.org/cmu_arctic/cmuarctic.data)

Please use the same format for utts.data as well for synthesizing new text data. Alternatively, you can use a text directory containing new text files (*.txt) to synthesize. You can't synthesize without the use of festival.

**Comment by maelp** [comment]
Thanks,

Do you mean that the downloaded data for `run_full_voice.sh` already has labels and alignments, and that when we want to use `merlin_synthesize.sh` we need to create those and this is why `festival` is used? Otherwise why would there be a need for different binaries to be installed? Could you tell me in what way `festival` is used when synthesizing data, and how the `run_full_voice.sh` synthesizes data without this?

I indeed used the same format as this file, but it did not work out-of-the-box, I will try on a Docker to see if this comes from osX

**Comment by ronanki** [comment]
Yes, the downloaded data for run_full_voice.sh already has labels and alignments. Festival was used to create labels and tools like ([Festvox/HTK](https://github.com/CSTR-Edinburgh/merlin/tree/master/misc/scripts/alignment)) were used to do the alignment. The data was then divided into three different sets -- train, valid, test and then synthesized few of those test files while running the script. 

Festival is required to create labels from text. Duration model is used for duration prediction and hence you don't require any other tools as part of synthesis. 

Festvox/HTK is required only if you want to do forced-alignment with new speaker's data or your own voice.
